app/controllers/concerns/load_settings.rb

```diff
@@ -18,6 +18,12 @@ def build_settings_defaults
         website_logo_url: '',
         website_url: ''
       })
+      SettingsObject.create(title: 'styles', values: {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.